packages/core/src/extensions/DraggableBlocks/DraggableBlocksPlugin.ts

```diff
@@ -22,14 +22,12 @@ let horizontalAnchor: number;
 let dragImageElement: Element | undefined;
 
 function getHorizontalAnchor() {
-  if (!horizontalAnchor) {
-    const firstBlockGroup = document.querySelector(
-      ".ProseMirror > [class*='blockGroup']"
-    ) as HTMLElement | undefined; // first block group node
-    if (firstBlockGroup) {
-      horizontalAnchor = absoluteRect(firstBlockGroup).left;
-    } // Anchor to the left of the first block group
-  }
+  const firstBlockGroup = document.querySelector(
```

Comment:
I don't think we need this function `getHorizontalAnchor` at all. in the mousemove listener, we already have the left position of the editor, and we can just use that I think?